client/src/components/StoreHeader/StoreHeader.tsx

```diff
@@ -11,9 +12,11 @@ function StoreHeader({
   link?: string | null;
```

Comment:
what does the link point to?

Reply:
this is a component assignable field for wherever we want to link the header button to. This will likely be depreciated in the near future as the use of history.push enables the browser/app back button to perform this function.